### DISCUSSION TITLE
LazyVim on Windows WSL not loading plugins folder.

### DISCUSSION THREAD

**Discussion by BraianPita** [LazyVim/LazyVim#4465] [Q&A] [answered]
I am trying to install my own custom plugins. I went through the docs and it mentions I just need to add a file to ~/.config/nvim/lua/*.lua and it should be automatically loaded by lazy.nvim. However, for some reason my plugins folder is not being imported by lazy.nvim. I don't see any of the plugins I am installing. I had issues installing vim-tmux-navigator. But then I just tried to use one of the recipes on the docs and it also would not load.

This is an example of a file not being used or not having any effect on my plugins folder:
```lua
return {
  {
    "folke/tokyonight.nvim",
    opts = {
      transparent = true,
      styles = {
        sidebars = "transparent",
        floats = "transparent",
      },
    },
  },
}
```

Or my tmux plugin:
```lua
return {
  {
    "alexghergh/nvim-tmux-navigation",
    lazy = false,
    priority = 1000,
    opts = {},
  },
}

```

I don't even see the tmux plugin on my Lazy UI `<leader>l`

**Comment by dpetka2001** [comment]
> I went through the docs and it mentions I just need to add a file to ~/.config/nvim/lua/*.lua

It should be `~/.config/nvim/lua/plugins/*.lua` instead. [Docs](https://www.lazyvim.org/configuration#-file-structure) also mention it.

**Reply by BraianPita** [reply, marked as answer]
I just figured out the issue. I accidentally messed up my `lua/config/lazy.lua` file. in such a way that I moved the `{import = "plugins"}` line to a different block accidentally, which broke my plugins folder without breaking anything else.... Talk about luck
